hudi-client/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -54,6 +52,11 @@
   // TODO: On by default. Once stable, we will remove the other mode.
   public static final String BLOOM_INDEX_BUCKETIZED_CHECKING_PROP = "hoodie.bloom.index.bucketized.checking";
   public static final String DEFAULT_BLOOM_INDEX_BUCKETIZED_CHECKING = "true";
+  public static final String BLOOM_INDEX_FILTER_TYPE = "hoodie.bloom.index.filter.type";
+  public static final String DEFAULT_BLOOM_INDEX_FILTER_TYPE = Integer.toString(BloomFilterTypeCode.SIMPLE.ordinal());
```

Comment:
could we keep it as a string.. much easier to configure than the ordinal?

Reply:
ok, sure. will fix it.